test/reputation-update.js

```diff
@@ -201,51 +197,39 @@ contract("Colony Reputation Updates", accounts => {
       await metaColony.addGlobalSkill(4);
       await metaColony.addGlobalSkill(5);
       await metaColony.addGlobalSkill(6);
-      const taskId1 = await setupRatedTask({
-        colonyNetwork,
-        colony: metaColony,
-        skill: 5
-      });
-      await metaColony.finalizeTask(taskId1);
+      await setupFinalizedTask({ colonyNetwork, colony: metaColony, skillId: 5 });
       let repLogEntryWorker = await inactiveReputationMiningCycle.getReputationUpdateLogEntry(3);
-      const result = toBN(WORKER_PAYOUT)
-        .muln(3)
-        .divn(2);
+      const result = toBN(WORKER_PAYOUT).muln(3).divn(2); // eslint-disable-line prettier/prettier
       assert.equal(repLogEntryWorker[1].toString(), result.toString());
       assert.equal(repLogEntryWorker[4].toNumber(), 6);
 
-      const taskId2 = await setupRatedTask({
-        colonyNetwork,
-        colony: metaColony,
-        skill: 6
-      });
-      await metaColony.finalizeTask(taskId2);
+      await setupFinalizedTask({ colonyNetwork, colony: metaColony, skillId: 6 });
       repLogEntryWorker = await inactiveReputationMiningCycle.getReputationUpdateLogEntry(7);
       assert.equal(repLogEntryWorker[1].toString(), result.toString());
       assert.equal(repLogEntryWorker[4].toNumber(), 8); // Negative reputation change means children change as well.
     });
 
     it("should revert on reputation amount overflow", async () => {
       // Fund colony with maximum possible int number of tokens
-      const maxUIntNumber = new BN(2).pow(new BN(255)).sub(new BN(1));
-      await fundColonyWithTokens(metaColony, colonyToken, maxUIntNumber);
-      // Split the tokens as payouts between the manager and worker
+      await fundColonyWithTokens(metaColony, colonyToken, INT256_MAX);
+
+      // Split the tokens as payouts between the manager, evaluator, and worker
       const managerPayout = new BN("2");
       const evaluatorPayout = new BN("1");
-      const workerPayout = maxUIntNumber.sub(managerPayout).sub(evaluatorPayout);
+      const workerPayout = INT256_MAX.sub(managerPayout).sub(evaluatorPayout);
```

Comment:
Why an int max rather than uint max?

Reply:
Because reputation is based on `ints`, so using `UINT256_MAX` would be interpreted as a negative value.

Reply:
Okay yes, it's just a badly named old variable, i.e. `maxUIntNumber`